** `mail-from-style' is set to 'system-default by default.
   Provided by: Romain Francoise <rfrancoise@debian.org>

---
 lisp/mail/sendmail.el |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

--- a/lisp/mail/sendmail.el
+++ b/lisp/mail/sendmail.el
@@ -58,7 +58,7 @@
   :type 'file)
 
 ;;;###autoload
-(defcustom mail-from-style 'default
+(defcustom mail-from-style 'system-default
   "Specifies how \"From:\" fields look.
 
 If `nil', they contain just the return address like:
